client/ayon_maya/api/lib_rendersettings.py

```diff
@@ -197,22 +197,14 @@ def _set_redshift_settings(self):
         # update the AOV list
         mel.eval("redshiftUpdateActiveAovList")
 
-        rs_p_engine = redshift_render_presets["primary_gi_engine"]
-        rs_s_engine = redshift_render_presets["secondary_gi_engine"]
-
-        if int(rs_p_engine) or int(rs_s_engine) != 0:
-            cmds.setAttr("redshiftOptions.GIEnabled", 1)
-            if int(rs_p_engine) == 0:
-                # reset the primary GI Engine as default
-                cmds.setAttr("redshiftOptions.primaryGIEngine", 4)
-            if int(rs_s_engine) == 0:
-                # reset the secondary GI Engine as default
-                cmds.setAttr("redshiftOptions.secondaryGIEngine", 2)
-        else:
-            cmds.setAttr("redshiftOptions.GIEnabled", 0)
-
-        cmds.setAttr("redshiftOptions.primaryGIEngine", int(rs_p_engine))
-        cmds.setAttr("redshiftOptions.secondaryGIEngine", int(rs_s_engine))
+        gi_enabled: bool = redshift_render_presets["gi_enabled"]
```

Comment:
```suggestion
        gi_enabled: bool = redshift_render_presets.get("gi_enabled", False)
```

Reply:
I am always so confused by this. The setting should be there because it's in the settings model.

Did you test without uploading the addon by any chance?

Reply:
> I am always so confused by this. The setting should be there because it's in the settings model.
> 
> Did you test without uploading the addon by any chance?

I tried to clean up the maya addon and re-install again. Working now. It could be because I am using development mode.

Reply:
@martastain @iLLiCiTiT do you know by any chance?

Say I deploy a new addon to the server with changed settings, would that new project settings directly get returned from the server when e.g. launching a new app. Or is there any reason to still fallback as if the setting would not exist?

If it were to update automatically, only thing I can think of why this may have failed for Kayla was running dev mode without having the revised addon uploaded to her server. (Do note that because in dev mode, it may of course be a  re-upload of an existing dev version).

In short, should this be `.get()` or can we rely on the keys existence through the Pydantic settings models?

Reply:
It can happen only in dev mode when you don't upload the addon to server. It won't happen in release...

Reply:
In that case, code should be fine as is. Thanks for confirming.